tools/@o3r/test-helpers/.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
`.eslintrc.cjs` should be enough, same for `.json`

Reply:
removed